client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -260,14 +260,13 @@ const RecurringPaymentsPlanAddEditModal = ( {
 						{ translate( 'Select renewal schedule' ) }
 					</FormLabel>
 					<FormSelect id="renewal_schedule" value={ editedSchedule } onChange={ onSelectSchedule }>
-						<option value="1 month">{ translate( 'Monthly' ) }</option>
-						<option value="1 year">{ translate( 'Yearly' ) }</option>
+						<option value="one-time">{ translate( 'One time sale (non-recurring)' ) }</option>
+						<option value="1 month">{ translate( 'Renew Monthly' ) }</option>
+						<option value="1 year">{ translate( 'Renew Yearly' ) }</option>
 					</FormSelect>
 				</FormFieldset>
 				<FormFieldset>
-					<FormLabel htmlFor="title">
-						{ translate( 'Please describe your subscription' ) }
-					</FormLabel>
+					<FormLabel htmlFor="title">{ translate( 'Please describe your product' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.